packages/babel-preset-env/src/utils.js

```diff
@@ -77,56 +95,97 @@ export const prettifyTargets = (targets: Targets): Object => {
     results[target] = value;
     return results;
   }, {});
-};
+}
 
-export const isUnreleasedVersion = (version: string, env: string): boolean => {
+export function isUnreleasedVersion(version, env) {
   const unreleasedLabel = unreleasedLabels[env];
   return (
     !!unreleasedLabel && unreleasedLabel === version.toString().toLowerCase()
   );
-};
+}
 
-export const getLowestUnreleased = (
-  a: string,
-  b: string,
-  env: string,
-): string => {
+export function getLowestUnreleased(a, b, env) {
   const unreleasedLabel = unreleasedLabels[env];
   const hasUnreleased = [a, b].some(item => item === unreleasedLabel);
   if (hasUnreleased) {
     return a === hasUnreleased ? b : a || b;
   }
   return semverMin(a, b);
-};
+}
 
-export const filterStageFromList = (list: any, stageList: any) => {
+export function filterStageFromList(list, stageList) {
   return Object.keys(list).reduce((result, item) => {
     if (!stageList[item]) {
       result[item] = list[item];
     }
 
     return result;
   }, {});
-};
+}
+
+export function isPolyfillSource(source) {
+  return source === "@babel/polyfill" || source === "core-js";
+}
+
+export function isBabelPolyfillSource(source) {
+  return source === "@babel/polyfill" || source === "babel-polyfill";
```

Comment:
I don't think that we should support `babel-polyfill` in v7

Reply:
This helper used only for deprecation message on `babel-polyfill` with `corejs: 3`.